themes/bootstrap3/templates/myresearch/profile.phtml

```diff
@@ -84,6 +84,42 @@
     <?php endif; ?>
   </div>
 
+  <?php $user = $this->auth()->isLoggedIn(); ?>
```

Comment:
Do you think it might be worth promoting this to a separate menu entry / controller action?

Reply:
I don't have strong feelings either way. I think it is small enough to justify its place in the profile menu and it makes sense to me to have saved logins in the same menu as "Change password" and other account actions.
Then again if it was a separate menu entry maybe it would be easier for the user to find their saved logins. And I guess saved logins are a bit different than basic account information. I'm fine with either way.

Reply:
I don't have strong feelings -- it just seems like we keep adding more and more things to the profile page and it's potentially getting quite long. But maybe all we need to do is introduce tabs or some equivalent navigation mechanism there if it gets too long.